docs/tutorials/start_index.md

```diff
@@ -4,15 +4,44 @@ The following documents will give you an overview of how to install and use
 napari. For more detailed use-cases, check out the [napari tutorials](./index)
 or [How-to guides](../howtos/index).
 
-- [napari quick start](./fundamentals/quick_start): This tutorial is for napari
-first-timers, to give them a quick glance of what napari does, and give it a try
-right away.
-- [How to install napari on your machine](./fundamentals/installation): This
-guide will teach you how to do a clean install of napari and launch the viewer.
-- [Getting started with napari](./fundamentals/getting_started): This tutorial
-will teach you all the different ways to launch napari.
-- [napari viewer tutorial](./fundamentals/viewer): This tutorial will teach you
-about the napari viewer, including how to use its graphical user interface (GUI)
+::::{grid}
+:::{grid-item-card} napari quick start
+:link: napari-quick-start
+:link-type: ref
+
+A quick glance of what napari does aimed at first-timers.
+:::
+::::
+
+::::{grid}
+:::{grid-item-card} How to install napari
+:link: installation
+:link-type: ref
+
+How to do a clean install of napari and launch the viewer.
+:::
+
+:::{grid-item-card} Getting started
+:link: getting-started
+:link-type: ref
+
+This tutorial will teach you all the different ways to launch napari.
+:::
+::::
+
+::::{grid}
+:::{grid-item-card} napari viewer tutorial
+:link: viewer-tutorial
+:link-type: ref
+
+This tutorial will teach you about the napari viewer, including how to use its graphical user interface (GUI)
 and how the data within it is organized.
-- [Layers at a glance](../guides/layers): An overview of the napari *Layers*,
-the basic viewable objects that can be added to a viewer.
\ No newline at end of file
+:::
+
+:::{grid-item-card} Layers at a glance
+:link: layers-glance
+:link-type: ref
+
+An overview of the napari *Layers*, the basic viewable objects that can be added to a viewer.
```

Comment:
```suggestion
An overview of the napari *Layers*, the basic viewable objects that can be displayed on the canvas.
```

I think this wording is much clearer. I know we have it as the former in the tutorial as well, but I'm a bit more ok with that since the whole doc then goes on to explain and give examples of what layers are.